docs/alphafeatures.md

```diff
@@ -26,9 +26,9 @@ The list of alpha features include:
 OpenEBS Mayastor developed using NVMe based architecture, targeted at addressing performance requirements of IO-intensive workloads is ready for alpha testing. For detailed instructions on how to get started with Mayastor please refer this [Quick-start guide](https://github.com/openebs/Mayastor/blob/develop/deploy/README.md).
 
 The following features are supported:
-- Creation of a Mayastor pool using block device attached to the node.
+- Creation of a Mayastor pool using block device attached to the node or by direct connection to existing iSCSI or NVMe-oF exports.
```

Comment:
@gila Do we need to mention `existing iSCSI volumes` instead of `existing iSCSI` ?

Reply:
Im not native english but I would argue no =)